main.go

```diff
@@ -108,9 +133,8 @@ func main() {
 		Development: true,
 	}
 	opts.BindFlags(flag.CommandLine)
-	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+	SetupLogging(&opts)
```

Comment:
```suggestion
	os.Args = append(os.Args, strings.Split(os.Getenv("ARGS"), " ")...)
	
	flag.Parse()

	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

If an option is appended multiple times only the last option value is applied, so we can simplify the `ARGS` parsing/appending.

Reply:
+1, will update my branch. thx.